### PR TITLE
UX: Review queue redesign fixes

### PR DESCRIPTION
**Pull request by chapoi**
Some styling changes + copy + reorganising of the actions

The delete options has been dismantled split up between agree/yes and ignore. 
~~The agree+keep option has been removed because it makes no sense.~~
Has been re-added for self-flagging purposes.

### Before:
image
image
image

### Now:
image
image
image

### CODE REVIEW

File: app/assets/javascripts/discourse/app/components/reviewable-score.hbs [hunk 1]
```diff
@@ -34,46 +27,50 @@
...
 </tr>
 
-{{#if this.rs.reason}}
-  <tr>
-    <td colspan="7">
-      <div class="reviewable-score-reason">{{html-safe this.rs.reason}}</div>
-    </td>
-  </tr>
-{{/if}}
+{{!-- {{#unless site.mobileView}}
```
Comment (ZogStriP) [inline app/assets/javascripts/discourse/app/components/reviewable-score.hbs:45]: Is this still needed?

Comment (chapoi) [reply app/assets/javascripts/discourse/app/components/reviewable-score.hbs]: Thank you, I isn't

**Comment by ZogStriP** [comment]
Looks like there are still some specs that are failing? @vinothkannans / @martin-brennan since you've both already worked on this PR, any chances either of you can have a quick look?

**Comment by martin-brennan** [comment]
@vinothkannans the failing plugin spec is because we have changed `perform_ignore` to `perform_ignore_and_do_nothing`. There was a reason around translations we named this IIRC, but it might be better to just switch back to `perform_ignore` if feasible? Otherwise you will need to make a PR for discourse-akismet and merge that with a discourse-compatibility change, since I don't think we can keep the old `build_action(:ignore)` in place since it will show in the UI too. The other JS test failing is just a flaky

**Comment by chapoi** [comment]
@vinothkannans Can I ask you to look at this/decide what to do based on what @martin-brennan said?

**Comment by vinothkannans** [comment]
@martin-brennan I added an action alias instead. The tests are passing now.

**Review by ZogStriP — APPROVED** [review]

**Pull request merged**